src/main/java/org/primeframework/mvc/sampleapp/action/BaseAction.java

```diff
@@ -1,8 +1,12 @@
 package org.primeframework.mvc.sampleapp.action;
 
+import org.primeframework.mvc.action.result.annotation.Forward;
 import org.primeframework.mvc.action.result.annotation.SaveRequest;
 
 // by default, this saves request and redirects the user to / if they are unauthenticated
-@SaveRequest(uri = "/")
+@SaveRequest(uri = "/login")
+@Forward.List(
+        @Forward(code = "error", page = "error.ftl", status = 500)
+)
```

Comment:
This is fine if we are trying to show an example of using `@Forward.List` - but if you just have a single `@Forward` the `.List` is not required.

Reply:
Done